components/layout.tsx

```diff
@@ -20,258 +23,41 @@ export const Layout: React.FC<React.PropsWithChildren<LayoutProps>> = ({
-                    </NavDropdown.Item>
-                    <NavDropdown.Item>
-                      <NavLink
-                        href="/why-use-maple/for-legislators"
-                        handleClick={closeNav}
-                      >
-                        {t("navigation.forLegislators")}
-                      </NavLink>
-                    </NavDropdown.Item>
-                  </NavDropdown>
-
-                  {authenticated && (
-                    <NavLink
-                      className={"navLink-primary"}
-                      handleClick={() => {
-                        closeNav()
-                        void signOutAndRedirectToHome()
-                      }}
-                    >
-                      {t("signOut", { ns: "auth" })}
-                    </NavLink>
-                  )}
-                </Nav>
-              </Navbar.Collapse>
-            </Navbar>
-          </NavBarBox>
-          <div
-            className={
-              isMobile
-                ? "me-2 w-100 h-100 flex"
-                : `col d-flex justify-content-start align-items-center`
-            }
-          >
-            <div
-              className={
-                isMobile
-                  ? `position-absolute top-0 start-50 z-1 translate-middle-x`
-                  : ""
-              }
-            >
-              <Nav.Link href="/" className="py-0 px-2 w-100">
-                {isMobile ? (
-                  <Image
-                    src="/Logo2024.png"
-                    alt="logo"
-                    className="w-100"
-                    width="60"
-                    height="60"
-                  />
-                ) : (
-                  <Image
-                    src="/Logo2024.png"
-                    alt="logo"
-                    width="80"
-                    height="80"
-                  />
-                )}
-              </Nav.Link>
-            </div>
-          </div>
-          <NavBarBox className={`justify-content-end`}>
-            <ProfileLink
-              role={claims?.role}
-              fullName={profile?.fullName}
-              isMobile={isMobile}
+    <>
+      {isClient ? (
+        <>
+          <Head>
+            <title>{formattedTitle}</title>
+            <link rel="icon" href="/favicon.ico" />
+          </Head>
+          <PageContainer>
+            {isMobile ? <MobileNav /> : <DesktopNav />}
+            <AuthModal />
+            <div className={`col`}>{children}</div>
+            <PageFooter
+              authenticated={authenticated}
+              user={user as any}
+              signOut={signOutAndRedirectToHome}
             />
-          </NavBarBox>
-        </NavBarBoxContainer>
-      </Container>
-
-      {isMobile && !authenticated ? (
-        <SignInWithButton className={`w-100`} />
-      ) : null}
-    </Navbar>
+          </PageContainer>
+        </>
+      ) : (
+        <>
+          <p>'This is never prerendered'</p>
```

Comment:
Do we expect this text to appear on dev? I'm seeing this text flash when re-rendering the page (e.g. by clicking the Logo in the mobile nav), which doesn't seem ideal

Reply:
I commented this out rather than delete it because it's a note regarding the code added to prevent hydration issues.  Note there in case better solutions are found so that the related isClient code can then been replaced.